DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+
+[=Media segments=] SHALL contain the media samples that exactly match the time span on the [=sample timeline=] that is assigned to the [=media segment=] by the MPD, except when using [=simple addressing=] in which case a certain amount of inaccuracy MAY be present as defined in [[#timing-addressing-inaccuracy]].
+
+Advisement: Even when using [=simple addressing=], the [=media segment=] that starts at or overlaps the [=period=] start point SHALL contain a media sample that starts at or overlaps the [=period=] start point and the [=media segment=] that ends at or overlaps the [=period=] end point SHALL contain a media sample that ends at or overlaps the [=period=] end point.
+
+## Segment addressing modes ## {#timing-addressing}
+
+This section defines the <dfn title="addressing mode">addressing modes</dfn> that can be used for referencing [=media segments=] and initialization segments in interopreable DASH presentations.
+
+Addressing modes not defined in this chapter SHALL NOT be used by DASH services. Clients SHOULD support all addressing modes defined in this chapter.
+
+You SHOULD choose the addressing mode based on the nature of the content:
+
+<dl class="switch">
+
+: Content generated on the fly
+:: Use [=explicit addressing=].
+
+: Content generated in advance of publishing
+:: Use [=explicit addressing=] or [=indexed addressing=].
+
+</dl>
+
+A service MAY use [=simple addressing=] which enables the packager logic to be very simple. This simplicity comes at a cost of reduced applicability to multi-period scenarios and reduced client compatibility.
+
+All [=representations=] in the same adaptation set SHALL use the same addressing mode. [=Representations=] in different adaptation sets MAY use different addressing modes. [[#timing-connectivity|Period-connected representations]] SHALL use the same addressing mode in every [=period=].
+
+### Indexed addressing ### {#timing-addressing-indexed}
+
+A representation that uses <dfn>indexed addressing</dfn> consists of an [[!ISOBMFF]] track file containing an index segment, an initialization segment and a sequence of [=media segments=].
+
+Clauses in section only apply to [=representations=] that use indexed addressing.
+
+Note: [[MPEGDASH]] makes a distinction between "segment" (HTTP-addressable entity) and "subsegment" (byte range of an HTTP-addressable entity). This document does not make such a distinction and has no concept of subsegments.
+
+<figure>
+	<img src="Images/Timing/IndexedAddressing.png" />
+	<figcaption>Indexed addressing is based on an index segment that references all [=media segments=].</figcaption>
+</figure>
+
+The MPD defines the byte range of the track file that contains the index segment. The index segment informs the client of all the [=media segments=] that exist, the time spans they cover on the [=sample timeline=] and their byte ranges.
+
+Multiple representations SHALL NOT be stored in the same track file.
```

Comment:
From @haudiobe: What does this mean? This is very confusing.

Reply:
I am experiencing difficulty in clarifying this as it seems pretty clear already. I request more details on the confusion. Does "CMAF track file" help? I add the word CMAF. Maybe the confusion is due to thinking of "representation" as files on disk (maybe some MPEG definition) rather than a sequence of segment references (the definition herein)?